README.md

```diff
@@ -1,8 +1,11 @@
 # Beaker
 
-Beaker is an acceptance testing harness for Puppet PE and other Puppet Projects.  It can also be used as a virtual machine provisioner - setting up machines, running any configuration on those machines and then exiting.
+Beaker is an acceptance testing harness for Puppet PE and other Puppet Projects.
```

Comment:
i suggest that is a SUT agnostic testing harness focused on interactions across multiple (virtual) machines (its core value add?), VM provisioning, and platform abstraction.

i'd like to see all references to Puppet as a product and Puppet Inc specific internals/infra removed from beaker (other than vmpooler, until the hypervisor stuff is decoupled).  pie in the sky?